eclipse/plugins/org.switchyard.tools.help/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <groupId>org.switchyard.tools</groupId>
+    <artifactId>switchyard-tools-eclipse-plugins</artifactId>
+    <version>2.4.0-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>org.switchyard.tools.help</artifactId>
+  <packaging>eclipse-plugin</packaging>
+  <name>JBoss SwitchYard Tooling ::  Plugins :: Help</name>
+
+   <build>
+     <plugins>
+       <plugin>
+       	 <groupId>org.apache.maven.plugins</groupId>
+         <artifactId>maven-antrun-plugin</artifactId>
+         <executions>
+           <execution>
+             <phase>package</phase>
+             <configuration>
+               <target>
+                 <!-- copy the generated plugin -->
+                 <copy todir="${basedir}" failonerror="false" overwrite="true">
+                   <fileset dir="${basedir}/../../rider-docs/output/eclipsehelp" excludes="META-INF/**" />
```

Comment:
that looks outdated. if thats something we have in fuseide too we should get rid of it